installer/pkg/common/constants.go

```diff
@@ -34,7 +34,7 @@ const (
 	RegistryAuthSecret          = "builtin-registry-auth"
 	RegistryTLSCertSecret       = "builtin-registry-certs"
 	RegistryFacadeComponent     = "registry-facade"
-	RegistryFacadeServicePort   = 3000
+	RegistryFacadeServicePort   = 30000
```

Comment:
Changed value as [registry-facade/podsecuritypolicy.go](https://github.com/gitpod-io/gitpod/blob/bdcae19a962677399ed58df79028df4847fb3a65/installer/pkg/components/registry-facade/podsecuritypolicy.go#L40-L43) set to `30000-33000`. It seems that this is post-processed in SaaS/Core Dev to be in that range

Reply:
Yes, that is done with Helm now [like so](https://github.com/gitpod-io/gitpod/blob/main/.werft/build.ts#L298-L302).